### PR TITLE
sjspielman authorshipt

### PR DESCRIPTION
**Pull request by sjspielman**
Addresses Issue #114 to add @sjspielman as author in the metadata.yaml.

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -352,6 +352,22 @@ authors:
...
+    initials: SJS
+    orcid: 0000-0002-9090-4788
+    twitter: stephspiel
+    email: spielman@rowan.edu
+    contributions:
+      - Validation - Miscellaneous code review
+      - Formal analysis - GSVA, some molecular subtyping
+      - Writing - Review and editing
+      - Visualization - Support for telomerase activity figures
+      - Supervision - Miscellaneous mentorship and oversight including survival analysis template and analyses
```
Comment (jaclyn-taroni) [inline content/metadata.yaml:366]:
```suggestion
      - Validation
      - Formal analysis
      - Writing - Review and editing
      - Visualization
      - Supervision
```

Can remove the specifics here

**Comment by sjspielman** [comment]
The build fail looks like an intermittent URL issue and not a real problem; I don't have sufficient permissions to trigger a re-build but I suspect a re-build will go through fine.

**Comment by jaclyn-taroni** [comment]
Let's see what happens when we commit my suggestion 🎲

**Review by jaclyn-taroni — APPROVED** [review]
✅

**Pull request merged**